__tests__/presets-loader.js

```diff
@@ -0,0 +1,72 @@
+
+const { describe, test, expect, beforeAll } = require('@jest/globals');
+const _ = require('lodash');
+
+const { loadConfigFromFixturePath } = require('./test-utils');
+
+
+describe('presets loader - model-files', () => {
+    let result;
+    beforeAll(async () => {
+        result = await loadConfigFromFixturePath('model-files');
+    });
+    
+    test('basic loading', () => {
+        expect(Object.keys(result.config.presets)).toMatchObject([
+            '.stackbit/presets/model1.json:presets[0]',
+            '.stackbit/presets/model1.json:presets[1]',
+            '.stackbit/presets/model1_another.json:presets[0]',
+            '.stackbit/presets/model1_another.json:presets[1]',
+            '.stackbit/presets/model_stackbit_1.json:presets[0]',
+            'node_modules/@stackbit/components/presets/model1.json:presets[0]',
+        ]);
+        expect(result.config.presets).toMatchObject(expect.objectContaining({
+            '.stackbit/presets/model1.json:presets[0]': {
+              label: 'my preset 1',
+              thumbnail: '.stackbit/presets/path/to/preset-thumbnail1.png',
+              data: {
+                'enum_field': 'thumbnail_1'
+              }
+            }
+        }));
+    });
+
+    test('inline model updating', () => {
+        expect(_.find(result.config.models, model => model.name === 'model_1')).toMatchObject(expect.objectContaining({
+            presets: [
+                '.stackbit/presets/model1.json:presets[0]',
+                '.stackbit/presets/model1.json:presets[1]',
+                '.stackbit/presets/model1_another.json:presets[0]',
+                '.stackbit/presets/model1_another.json:presets[1]',
+                'node_modules/@stackbit/components/presets/model1.json:presets[0]',
+            ]
+        }));
+    });
+
+    test('thumbnail resolving', () => {
+        expect(result.config.presets).toMatchObject(expect.objectContaining({
+            '.stackbit/presets/model1.json:presets[0]': expect.objectContaining({
+                thumbnail: '.stackbit/presets/path/to/preset-thumbnail1.png'
+            })
+        }));
+        expect(result.config.presets).toMatchObject(expect.objectContaining({
+            '.stackbit/presets/model1.json:presets[1]': expect.objectContaining({
+                thumbnail: '/images/preset-thumbnail2.png'
```

Comment:
After thumbnails paths are resolved they all should be relative to project root, and without `/` prefix.
This is also because if we run `path.resolve(projDir, thumbnail)` it should return correct path. And `path.resolve` method will always ignore the `projDir` if `thumbnail` starts with `/`.